32 update readme for m1 users (#33)
mexca/mexca: README.md

conda create -n mexca-venv
conda activate mexca-venv
```
Once you have activated your virtual environment (mexca-venv) you can then install mexca from PyPi:

```console
python3 -m pip install mexca
```

To install mexca from the GitHub repository, do:

```console
git clone https://github.com/mexca/mexca.git
cd mexca
python3 -m pip install .
```

Or via:

```console
python3 -m pip install git+https://github.com/mexca/mexca.git
```

### Installation steps on Unix/macOS

> Open the terminal (Click the Launchpad icon in the Dock, type Terminal in the search field, then click Terminal. Otherwise, you can use the keyboard shortcut Command + Space, and type in “Terminal”).

Reviewer: ```suggestion
Open the terminal (click the Launchpad icon in the Dock, type “Terminal” in the search field; otherwise, you can use the keyboard shortcut `Command` + `Space`, and type in “Terminal”).
```

python3 -m pip install git+https://github.com/mexca/mexca.git
```

#### Issues installing mexca for M1 Macbook users

> Many ML libraries (that we import in Mexca) do not fully support Apple M1 yet, which leads to several issues in installing mexca. We provide below few workarounds for the most common issues. Note that the following fixes have been tested on python 3.9.0 in a conda environment, and they are dated back to 3rd of October 2022.

Reviewer: ```suggestion
Many deep learning libraries that we import in mexca do not fully support the Apple M1 yet, which can lead to several issues when installing mexca. We provide few workarounds for the most common issues. They have been tested on Python 3.9.0 in a conda environment (last update 3/10/2022).
```

Error n. 1: 

- OSError cannot load libsndfile.dylib (Github issue [#311](https://github.com/bastibe/python-soundfile/pull/311)):

```console
OSError: cannot load library '...venv/lib/python3.9/site-packages/_soundfile_data/libsndfile.dylib': dlopen(...venv/lib/python3.9/site-packages/_soundfile_data/libsndfile.dylib, 2): image not found
```

To fix this:

> (1) Make sure that you have installed libsndfile via brew, if not [install it](https://formulae.brew.sh/formula/libsndfile). 
> 
> (2) copy the libsndfile installed from Homebrew (/opt/homebrew/lib/_soundfile_data/libsndfile.dylib) into the expected folder ‘python3.9/site-packages/_soundfile_data/‘ 
> 
> (3) Restart the kernel

Reviewer: ```suggestion
1. Make sure that you have installed libsndfile via brew, if not [install it](https://formulae.brew.sh/formula/libsndfile). 
2. Copy the libsndfile installed from Homebrew (/opt/homebrew/lib/_soundfile_data/libsndfile.dylib) into the expected folder ‘python3.9/site-packages/_soundfile_data/‘ 
3. Restart the kernel.
```

This formats the numbered bullets points.

Error n. 2: 

- OSError cannot load libllvmlite.dylib (Github issue [#650](https://github.com/numba/llvmlite/issues/650)):

```console
OSError: Could not load shared object file: libllvmlite.dylib
```

> To fix this,

Reviewer: ```suggestion
To fix this:
```

> (1) type in the terminal:

Reviewer: ```suggestion
1. Type in the terminal:
```

```console
conda install -c numba numba
conda install -c numba llvmlite
```

> (2) Restart the kernel

Reviewer: ```suggestion
2. Restart the kernel.
```
